src/index.html

```diff
@@ -68,7 +68,7 @@ <h4>Make your server how you want it.</h4>
                     <a href="http://www.minecraftforge.net">Minecraft Forge</a>. We're also able to respond to Minecraft updates quickly, so you'll never have an
                     excuse to not update.
                 </p>
-                <a href="#!" class="waves-effect waves-light btn light-blue darken-2"><i class="material-icons left">archive</i>Learn more about Velocity's server support</a>
+                <a href="https://docs.velocitypowered.com/en/latest/users/player-info-forwarding.html" class="waves-effect waves-light btn light-blue darken-2"><i class="material-icons left">archive</i>Learn more about Velocity's server support</a>
```

Comment:
https://docs.velocitypowered.com/en/latest/users/frequently-asked-questions.html#what-server-software-is-supported-by-velocity would be better to link here.